BotProject/Templates/CSharp/Program.cs

```diff
@@ -42,11 +42,6 @@ public static void Main(string[] args)
                     builder.AddJsonFile("appsettings.deployment.json", optional: true, reloadOnChange: true);
                 }
 
-                if (!env.IsDevelopment())
```

Comment:
why are you removing this?

Reply:
currently we don't have an user secret inside the botruntime, app password and luis authoring key are stored in file(appsettings)

Reply:
> why are you removing this?

currently we don't have an user secret inside the botruntime, app password and luis authoring key are stored in file(appsettings)